app/src/main/java/net/skyscanner/backpack/demo/compose/BottomSheetStory.kt

```diff
@@ -52,13 +62,66 @@ import net.skyscanner.backpack.demo.ui.ListItem
 fun BottomSheetStory(
     modifier: Modifier = Modifier,
     initialValue: BpkBottomSheetValue = BpkBottomSheetValue.Collapsed,
+    dragHandleStyle: BpkDragHandleStyle = BpkDragHandleStyle.Default,
```

Comment:
as there's a separate story for the image we shouldn't need the drag handle here anymore?

Reply:
Removed!